scripts/update-tezos.sh

```diff
@@ -30,6 +30,9 @@ our_tezos_tag="$(jq -r '.tezos_ref' meta.json | cut -d'/' -f3)"
 new_meta=$(jq ".tezos_ref=\"$latest_upstream_tag\"" meta.json)
 echo "$new_meta" > meta.json
 
+# Update version of tezos-baking package
+sed -i "s/version = .*/version = \"$latest_upstream_tag\"/" ./code/pyproject.toml
```

Comment:
IMO this should be moved a few lines below and be performed with the `release` `sed` command, so that it's committed with it.